pkg/data_utils/data_utils.go

```diff
@@ -0,0 +1,58 @@
+/*
+This file is part of REANA.
+Copyright (C) 2022 CERN.
+
+REANA is free software; you can redistribute it and/or modify it
+under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+// Package data_utils gives extra functions to manipulate data structures like strings and slices.
```

Comment:
Cosmetics: Please rename `data_utils` to `datautils` or `datautil` everywhere, i.e. without underscore. Let's try to keep the package name to one word only.  Ditto for `error_handler` -> `errorhandler`.